alcatel_sros_ping.textfsm

```diff
@@ -0,0 +1,14 @@
+Value SENT_QTY (\d+)
+Value DESTINATION (\d+.\d+.\d+.\d+)
+Value LOSS_PCT (\d+)
```

Comment:
Please check any existing `ping` templates to make sure we align with the value names.

Also, please include `.raw` and `.yml` files within the tests. Our README has more details on contributing to this project.